paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -630,15 +630,22 @@ def times_up():
                     human_max_deploy_time = humanize.naturaldelta(
                         datetime.timedelta(seconds=self.timeout)
                     )
+                    stuck_bounce_runbook = os.environ.get(
+                        "STUCK_BOUNCE_RUNBOOK", DEFAULT_STUCK_BOUNCE_RUNBOOK,
+                    )
```

Comment:
is this so that jenkins can set this?

Reply:
No, more because paasta is open-source, so I wanted to have at least a way to change it externally.